mantle/kola/harness.go

```diff
@@ -549,15 +551,44 @@ func runProvidedTests(tests map[string]*register.Test, patterns []string, multip
 			reporters.NewJSONReporter("report.json", pltfrm, versionStr),
 		},
 	}
+
+	var confs []*conf.Conf
```

Comment:
Maybe `nonExclusiveTestsConf`?

Reply:
Feels like here we can just do the splitting of non-exclusive tests into `tests` and `nonExclusiveTests` and do the `RenderToV34exp` and merge inside `makeNonExclusiveTest`.